fastq_filtrator.py

```diff
@@ -0,0 +1,39 @@
+def gc_share(string):  # returns CG content in one string in percent
+    return round((string.count('c') + string.count('C') + string.count('g') + string.count('G')) / len(string) * 100, 1)

```

Comment:
1. Зачем округлять GC?
2. Можно сделать str.upper и считать колтчество букв 2 раза вместо одного
3. Это хорошо читается, но не очень оптимально по производительности. count внутри себя делает полный проход по строке, и мы вызываем его аж четыре раза, хотя GC можно посчитать за один проход. Но это мелочи)
```suggestion
    string = string.upper()
    return (string.count('C') + string.count('G')) / len(string) * 100

```